addons/gamification/views/karma_ranks_views.xml

```diff
@@ -0,0 +1,58 @@
+<odoo>
+    <data>
+        <!-- Ranks views -->
+        <record id="gamification_karma_ranks_action" model="ir.actions.act_window">
+            <field name="name">Ranks</field>
+            <field name="res_model">gamification.karma.rank</field>
+            <field name="view_mode">tree,form</field>
+            <field name="help" type="html">
+                <p class="o_view_nocontent_smiling_face">
+                    Create a new rank
+                </p><p>
+                    A rank correspond to a fixed karma level. The more you have karma, the more your rank is high.
+                    This is used to quickly know which user is new or old or highly or not active.
+                </p>
+            </field>
+        </record>
+
+        <record id="gamification_karma_ranks_view_tree" model="ir.ui.view">
+            <field name="name">Ranks List</field>
```

Comment:
Guidelining: name of views (not actions) is just xml id with '. instead of '_'' so that it is standardized.